subprojects/core-api/src/main/java/org/gradle/api/artifacts/ProjectDependency.java

```diff
@@ -28,11 +28,18 @@
 @HasInternalProtocol
 @SuppressWarnings("deprecation") // Because of SelfResolvingDependency
 public interface ProjectDependency extends ModuleDependency, SelfResolvingDependency {
+
     /**
-     * Returns the project associated with this project dependency.
+     * Get the path to the project that this dependency refers to relative to its owning build.
      *
-     * @deprecated This method will be removed in Gradle 9.0. Accessing the mutable
-     * state of other projects should be avoided.
+     * @see Project#getPath()
+     *
+     * @since 8.11
+     */
+    String getPath();
+
+    /**
+     * Returns the project associated with this project dependency.
```

Comment:
Don't we still want something like the `@deprecated` part? e.g., 
> This method will be removed in Gradle 9.0.

Reply:
Ah yeah will add this back